stistools/defringe/prepspec.py

```diff
@@ -1,13 +1,19 @@
 #! /usr/bin/env python
 
 import os
+import re
 import numpy as np
+import argparse
+import textwrap
 from astropy.io import fits
+import stistools
```

Comment:
Use relative imports:
`from ..calstis import calstis`

This won't work when running the file directly, but can be tested by `python install python.py` and the importing the code in the interpreter.

Reply:
Resolved in my push.